src/conn/conn_reconfig.c

```diff
@@ -25,9 +26,57 @@ __wt_conn_compat_config(WT_SESSION_IMPL *session, const char **cfg)
 	if (cval.len == 0) {
```

Comment:
It's a pity this parsing logic is duplicated for compatability.release and compatibility.require_min - can you see a path to sharing code?

Reply:
Fixed.